lib/charms/operator_libs_linux/v0/dnf.py

```diff
@@ -0,0 +1,462 @@
+#!/usr/bin/env python3
```

Comment:
I add the `#!/usr/bin/env python3` hashbang to all my files so that I know what kind of file I am working with when reading outside of a Python-dedicated editor. I can remove it if we feel like it is necessary to be removed.

Reply:
Yeah, I think it sends the wrong signal. The hash bang says how to execute the script as a stand-alone executable (but it's not intended to be executable). Isn't the `.py` extension enough of a signal for "what kind of file" it is? The file extension is how editors normally auto-detect file type.